R/constants.R

```diff
@@ -18,7 +18,7 @@ FLAG_SUM_TEST   <- "FLAG_SUM_TEST"
 # Time constants
 HISTORICAL_YEARS <- 1971:2010
 FUTURE_YEARS <- seq( 2015, 2100, 5 )
-
```

Comment:
@bpbond yes, i agree.  @CLynchy can you put this under the emissions constants section?